### ISSUE TITLE
Example create_terrain() call fails on action()

### ISSUE THREAD

**Issue by wjones127** [ropensci/unifir#6]
I attempted to create a terrain with [the example](https://github.com/mikemahoney218/unifir/blob/2e5f90cfc09b4456b3751f932f19da49e0dbebc7/R/create_terrain.R#L14-L35), but it reported there was an exception. Is there a way to see that exception?

```r
library(unifir)

raster <- tempfile(fileext = ".tiff")
r <- terra::rast(matrix(rnorm(1000^2, mean = 100, sd = 20), 1000),
                 extent = terra::ext(0, 1000, 0, 1000)
)
terra::writeRaster(r, raster)


script <- make_script(project = "projects/example_script") |>
  add_light(light_type = "Directional", light_name = "Sun",
            x_rotation = 35, y_rotation = 20) |>
  add_default_player() |>
  create_terrain(
    method_name = "MakeTerrain",
    heightmap_path = raster,
    x_pos = 0,
    z_pos = 0,
    width = 1000,
    height = terra::minmax(r)[[2]],
    length = 1000,
    heightmap_resolution = 1000
  ) |>
  save_scene(scene_name = "my_scene") |>
  set_active_scene(scene_name = "my_scene")

action(script)
# Aborting batchmode due to failure:
# executeMethod method MakeTerrain.MainFunc threw exception.
#> Error in action(script): 1
```

**Comment by mikemahoney218** [comment]
Thanks! Fixed in bfbb306 -- I had been testing with local files and mistakenly assumed that I could swap the local path for a tempfile (for CRAN) with no problems.

Unfortunately, that's actually the entire error message provided by Unity when called on the command line. There might be a space for a vignette on how to debug these scripts (which is a bit more about the Unity engine and C# than anything to do with R) but Unity's CLI is notoriously unfriendly, so I don't believe there's any way to surface these issues within R itself.

**Comment by wjones127** [comment]
Can confirm that this example now works in that fix branch.

> Unity's CLI is notoriously unfriendly, so I don't believe there's any way to surface these issues within R itself.

That's a shame 😢
